src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserControllerIntTest.kt

```diff
@@ -211,26 +215,32 @@ class UserControllerIntTest : IntegrationTestBase() {
       val uuid = UUID.randomUUID()
       externalUsersApiMockServer.stubUserByUsername(username)
       hmppsAuthMockServer.stubUserIdByUsernameAndSource(username, auth, uuid)
+
       webTestClient
         .get().uri("/users/me")
         .headers(
           setAuthorisation(),
         )
         .exchange()
         .expectStatus().isOk
-        .expectBody()
-        .jsonPath("$").value<Map<String, Any>> {
-          assertThat(it).containsExactlyInAnyOrderEntriesOf(
-            mapOf(
-              "username" to "AUTH_ADM",
-              "active" to true,
-              "name" to "Ext Adm",
-              "authSource" to "auth",
-              "userId" to "5105a589-75b3-4ca0-9433-b96228c1c8f3",
-              "uuid" to uuid.toString(),
-            ),
-          )
-        }
+
+      println("........ and again ............")
+      println("...............................")
+
+      webTestClient
+        .get().uri("/users/me")
+        .headers(
+          setAuthorisation(),
+        )
+        .exchange()
+        .expectStatus().isOk
+
+      WireMock(8090).find(RequestPatternBuilder.allRequests()).forEach(
+        Consumer { request: LoggedRequest ->
+          println(request.absoluteUrl)
```

Comment:
WITHOUT the expires_in set in the token response, WireMock reports the following:
```
http://localhost:8090/auth/oauth/token
http://localhost:8090/auth/oauth/token
http://localhost:8090/auth/api/user/AUTH_ADM/auth
http://localhost:8090/auth/oauth/token
http://localhost:8090/auth/oauth/token
http://localhost:8090/auth/api/user/AUTH_ADM/auth
```

WITH the expires_in set to something large like 3600, we get:

```
http://localhost:8090/auth/oauth/token
http://localhost:8090/auth/oauth/token
http://localhost:8090/auth/api/user/AUTH_ADM/auth
http://localhost:8090/auth/api/user/AUTH_ADM/auth
```

So I think this is proving that it SHOULD be caching the token 🤔 As long as the expires_in is long enough.  Worth checking this tomorrow?